src/types/AdInfo.ts

```diff
@@ -151,33 +178,6 @@ export type AdRewardInfo = AdInfo & {
     rewardAmount: string;
 };
 
-/**
```

Comment:
Normally, we should deprecate this first as to not break existing integrations if they update the plugin, but I guess it is ok for something seldom used